src/app/core/services/issue.service.ts

```diff
@@ -430,6 +429,18 @@ export class IssueService {
     return this.issueCommentService.getIssueComments(issueId, this.isIssueReloaded).pipe(
       map((issueComments: IssueComments) => {
         const issueComment = this.getIssueComment(issueComments);
+        let teamResponse;
+        if (!!issueComment) {
```

Comment:
I think this chunk of code can be :
`let teamResponse = issueInJson['teamResponse'];`
` if (!!issueComment && currentPhase === phaseTesterResponse) { teamReponse =.. }`
` else if (!!issueComment && currentPhase === phaseTeamResponse { teamResponse = ...}` 

to reduce nesting, what do you think?

Reply:
Sounds good 👍